src/main/java/de/tum/in/www1/artemis/service/connectors/localci/buildagent/LocalCIDockerService.java

```diff
@@ -101,6 +101,9 @@ public void deleteOldDockerImages() {
             return;
         }
 
+        // Get map of docker images and their last build dates
+        IMap<String, ZonedDateTime> dockerImageCleanupInfo = hazelcastInstance.getMap("dockerImageCleanupInfo");
```

Comment:
Idea for a followup / optional:

We should declare these names somewhere as constants to avoid issues with typos / copy paste errors or even potential name clashes.

Reply:
good point, I'll keep that in mind